File: python/plusMinus.py
arr = [6,[-4,3,-9,0,4,1]]
def plusMinus(arr):
    count = 0
    positive = 0
    negative = 0
    cero = 0
    while count < arr[0]:
        if arr[1][count] > 0:
            positive +=1
        if arr[1][count] < 0:
            negative +=1
        if arr[1][count] == 0:
            cero +=1
        count += 1
    # print(positive)
    # print(negative)
    # print(cero)
    print(format(positive/arr[0],'.6f'))
    print(format(negative/arr[0],'.6f'))
    print(format(cero/arr[0],'.6f'))
plusMinus(arr)

File: js/plusMinus.js
const input = [ 6,
    -4, 3, -9, 0, 4, 1, 
];

function plusMinus(arr){
    let poitive = 0;
    let negative = 0;
    let cero = 0;
    for(let i = 1; i <= arr[0]; i++){
        if(arr[i] > 0){
            poitive++;
        }
        if(arr[i] < 0){
            negative++;
        }
        if(arr[i] === 0){
            cero++;
        }  
     
    }
    console.log(parseFloat(poitive/arr[0]).toFixed(6));
    console.log(parseFloat(negative/arr[0]).toFixed(6));
    console.log(parseFloat(cero/arr[0]).toFixed(6));
}
plusMinus(input);